tests/ingestIndexTestBase.py

```diff
@@ -224,6 +227,10 @@ def setUpClass(cls):
     def setUp(self):
         self.repoPath = tempfile.TemporaryDirectory()  # cleaned up automatically when test ends
         self.butler = self.makeTemporaryRepo(self.repoPath.name, self.depth)
+        self.logger = lsst.log.Log.getLogger('lsst.ReferenceObjectLoader')
```

Comment:
Why is this an LSST logger? I've been switching things away from that. Assert logs will work without the forwarding because all task loggers are now python loggers.  What am I missing?

Reply:
I did it that way it because I didn't know we could just use `logging.getLogger` for this kind of thing. I've switched it.